interactions/api/models/channel.py

```diff
@@ -164,3 +166,139 @@ def __init__(self, **kwargs):
             if self._json.get("last_pin_timestamp")
             else None
         )
+
+    async def send(
+        self,
+        content: Optional[str] = None,
+        *,
+        tts: Optional[bool] = False,
+        # attachments: Optional[List[Any]] = None,  # TODO: post-v4: Replace with own file type.
+        embeds=None,
+        allowed_mentions=None,
+        components=None,
+    ):
+        """
+        Sends a message in the channel
+
+        :param content?: The contents of the message as a string or string-converted value.
+        :type content: Optional[str]
+        :param tts?: Whether the message utilizes the text-to-speech Discord programme or not.
+        :type tts: Optional[bool]
+        :param embeds?: An embed, or list of embeds for the message.
+        :type embeds: Optional[Union[Embed, List[Embed]]]
+        :param allowed_mentions?: The message interactions/mention limits that the message can refer to.
+        :type allowed_mentions: Optional[MessageInteraction]
+        :param components?: A component, or list of components for the message.
+        :type components: Optional[Union[Component, List[Component]]]
+        :return: The sent message as an object.
+        :rtype: Message
+        """
+        from ...models.component import ActionRow, Button, SelectMenu
+        from .message import Message
+
+        _content: str = "" if content is None else content
+        _tts: bool = False if tts is None else tts
+        # _file = None if file is None else file
+        # _attachments = [] if attachments else None
+        _embeds: list = (
```

Comment:
This `_embeds` value declaration falls out of line for most of the conditionals we create for type-based values.

```py
_embeds: list = []
ifisinstance(embeds, list):
    _embeds = [embed._json for embed in embeds]
else:
    _embeds = [embeds._json]